src/chrome/content/rules/ykimg.com.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+	Invalid certificate:
+		res.mfs.ykimg.com
+-->
+
+<ruleset name="ykimg.com">
+	<target host="css.ykimg.com" />
+		<test url="http://css.ykimg.com/youku/dist/css/find/external/ykheader_37.css" />
+	<target host="g1.ykimg.com" />
+		<test url="http://g1.ykimg.com/05110000494903FC6730B3562DD73071" />
+	<target host="g2.ykimg.com" />
+		<test url="http://g2.ykimg.com/05110000494903FC6730B3562DD73071" />
+	<target host="g3.ykimg.com" />
+		<test url="http://g3.ykimg.com/05110000494903FC6730B3562DD73071" />
+	<target host="g4.ykimg.com" />
+		<test url="http://g4.ykimg.com/05111A914EFE8CAE6708BE7CB0FA043D" />
```

Comment:
404 for me, maybe replace with http://g4.ykimg.com/01270F1F464E5872635C1600000000122A1C24-0703-6A13-619E-F6618C1E8751?